strings/french.json

```diff
@@ -255,7 +255,7 @@
   "Income Details": "Détails de Revenu",
   "No, I don't": "Non",
   "Oops, you entered a negative number": "Oups, vous avez entré un nombre négatif",
-  "The amount cannot have more than 2 decimals": "Le montant ne peut pas avoir plus de deux décimales",
+  "The amount must be a number (e.g. 100.75)": "Le montant doit être un nombre (par exemple 100,75)",
```

Comment:
Leave this unmodified, the `.json` is created from the `.strings` file, which is where the translation should happen, and the `.strings` in turn are created from the UI (`i18n` tag) or JS (`L` function). Then run the `strings` command per the [`strings` readme](https://github.com/okTurtles/strings/blob/master/README.md#help-translate-a-language).

Reply:
Ok. My intent was to emulate the effects of running the `strings` tool so as to avoid the need for someone else to run it, since I'm unfortunately still on an unsupported platform.

Reply:
Oh, this was actually OK. My bad, I shouldn't have written that comment, because you're right, you can actually modify these files directly like this. The only reason why you might not want to do it manually is that there might be some format change or some other automated thing that the strings command will do differently. Anyway, you are free to un-revert this change, or we'll run the strings command ourselves later. Apologies, today has been a long day and my brain clearly wasn't thinking straight when it came to this.

Reply:
No problem, I needed some `git restore` practice anyway 😄